tests/templates/test_subroutines/test_hilbert_schmidt.py

```diff
@@ -301,32 +243,6 @@ def v_circuit(params):
             assert i.data == j.data
             assert i.wires == j.wires
 
-    def test_lhs_adjoint_method_1_qubit(self):
```

Comment:
Is this removed because the principle is tested elsewhere?

Reply:
Now relying on default behaviour instead of in-place inversion.